superbench/analyzer/diagnosis_rule_op.py

```diff
@@ -95,16 +97,18 @@ def variance(data_row, rule, summary_data_row, details, categories):
                 violate_metric = eval(rule['criteria'])(var)
                 # add issued details and categories
                 if violate_metric:
-                    pass_rule = False
+                    label_metric_num += 1
                     info = '(B/L: {:.4f} VAL: {:.4f} VAR: {:.2f}% Rule:{})'.format(
                         baseline, val, var * 100, rule['criteria']
                     )
                     details.append(metric + info)
                     categories.add(rule['categories'])
-        return pass_rule
+        if 'store' in rule and rule['store'] is True:
+            label[rule['name']] = label_metric_num
```

Comment:
I think you need to return `label`?

Reply:
It's not a right way to change the label as a pointer.

Reply:
is it ok to return violated_metric_num